Nargo.toml

```diff
@@ -5,6 +5,7 @@ members = [
     "ethereum_history_api/circuits/get_storage",
     "ethereum_history_api/circuits/get_account",
     "ethereum_history_api/circuits/get_header",
-    "vlayer/ethereum/circuit/lib"
+    "vlayer/ethereum/circuits/lib",
+    "vlayer/ethereum/circuits/is_dao_worthy",
```

Comment:
I propose to put it under `valyer/examples/circuits/is_dao_worthy `

Reply:
OK, moved to `valyer/examples/circuits/is_dao_worthy ` directory